CHANGELOG.md

```diff
@@ -31,6 +31,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * `RepoPath` template type now has a `absolute() -> String` method that returns
   the absolute path as a string.
 
+* `jj workspace root` now accepts a `<WORKSPACE>` argument to show the root path
+  of the specified workspace (defaults to the current one).
```

Comment:
Since `jj workspace root` will throw error on workspaces created before this change, do I add that information in breaking changes?

Reply:
The second one, please.

Also, I think the optional workspace argument should be a named argument instead of a positional one (so e.g. `jj workspace root --workspace bar`).

Thanks